src/styles/blocks/modifiers.scss

```diff
@@ -0,0 +1,18 @@
+@for $day-number from 1 through 31 {
+  .calendar__day:nth-child(#{$day-number})::before {
+    content: '#{$day-number}';
+    color: $black-color;
+    padding-left: 1px;
+  }
+
+  .calendar--month-length-#{$day-number}
+    .calendar__day:nth-child(-n + #{$day-number}) {
+    display: flex;
```

Comment:
The same applies here. Consider using a CSS variable for the month length instead of generating 31 classes. This would reduce the size of the CSS file and make the code more maintainable.